knack/iframeMapMessenger.js

```diff
@@ -79,9 +72,12 @@
       if (data.message === "LAT_LON_FIELDS") {
```

Comment:
could this have a more descriptive name? e.g. `LAT_LON_UPDATE` or `SIGN_LOCATION_UPDATE`